### ISSUE TITLE
adapt_framework: Disabled submit button should be available to screen reader

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#2950]
### Subject of the issue
The user is unaware of the existence of a button as `button[disabled]` elements are not available to the screen reader.
